sklearn/ensemble/_hist_gradient_boosting/tests/test_loss.py

```diff
@@ -80,20 +84,22 @@ def fprime2(x):
 
 @pytest.mark.parametrize('loss, n_classes, prediction_dim', [
     ('least_squares', 0, 1),
+    ('least_absolute_deviation', 0, 1),
     ('binary_crossentropy', 2, 1),
     ('categorical_crossentropy', 3, 3),
 ])
 @pytest.mark.skipif(Y_DTYPE != np.float64,
                     reason='Need 64 bits float precision for numerical checks')
-def test_numerical_gradients(loss, n_classes, prediction_dim):
+@pytest.mark.parametrize('seed', range(1))
```

Comment:
O_o

Reply:
^^

I can remove it, it's just convenient when testing locally